src/ucp/wireup/wireup.c

```diff
@@ -280,6 +280,10 @@ static double ucp_runtime_score_func(ucp_worker_h worker, uct_iface_attr_t *ifac
                  UCT_IFACE_FLAG_ATOMIC_SWAP64 | UCT_IFACE_FLAG_ATOMIC_CSWAP64;
     }
 
+    if (context->config.features & UCP_FEATURE_WAKEUP) {
+        flags |= UCT_IFACE_FLAG_WAKEUP;
```

Comment:
this may be a problem, do we really want to disqualify all transports which do not support wakeup? can there be a better way?